scalableminds/webknossos
# Min length for layer names is one

### Steps to test:
- Go to dataset list view > Settings  and change the dataset name in the form to less then 3 characters. There should be a form error. 
- In the same form, edit a layer name. You should be able to set a single character name.
- Add a remote dataset. Repeat steps 1 and 2 in the second form (after clicking `Add Layer`).

### Issues:
- fixes #7020

------
(Please delete unneeded items, merge only when none are left open)
~~- [ ] Updated [migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable~~
~~- [ ] Updated [documentation](../blob/master/docs) if applicable~~
~~- [ ] Adapted [wk-libs python client](https://github.com/scalableminds/webknossos-libs/tree/master/webknossos/webknossos/client) if relevant API parts change~~
~~- [ ] Needs datastore update after deployment~~
